db/seeds.rb

```diff
@@ -1,125 +1,187 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Admin.create([{ first_name: "Admin_first",
+								last_name: "Admin_last",
+								email: "admin@email.com",
+								password: "password"
+								}])
+
+User.create([{ first_name: "Alice",
+							 last_name: "alast_name",
+						   email: "alice@email.com",
+						   display_name: "user_1",
+						   password: "password"},
+						  {first_name: "Bob",
+						   last_name: "blast_name",
+						   email: "bob@email.com",
+						   display_name: "user_2",
+						   password: "password"},
+						  {first_name: "Carol",
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space inside { missing.<br>Tab detected.